docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md

```diff
@@ -1,32 +1,40 @@
 ---
 id: 1password-source
+slug: /c2c/1password
 title: 1Password Source
```

Comment:
@JV0812 , to make it consistent with the other C2Cs, I would suggest adding the sidebar_label to be "1Password". What do you think?